### PR TITLE
ADM-350 Pipeline tool section

### PR DESCRIPTION
**Pull request by spadio9587**
## Summary
check with backend and confirm the URL and add buildKite url in application yml
...

## Before

_Description_

Before

## After

_Description_

After

## Note

_Null_

### CODE REVIEW

**Review by Teresasasa — APPROVED** [review]
LGTM

File: frontend/src/clients/PipelineToolClient.ts [hunk 1]
```diff
@@ -16,7 +16,9 @@ export class PipelineToolClient extends HttpClient {
 
   verifyPipelineTool = async (params: getVerifyPipelineToolParams) => {
     try {
-      const result = await this.axiosInstance.get('/pipeline/fetch', { params: { ...params } }).then((res) => res)
+      const result = await this.axiosInstance
+        .get(`/pipelines/${params.type}`, { params: { ...params } })
+        .then((res) => res)
```
Comment (guzhongren) [inline frontend/src/clients/PipelineToolClient.ts:21]: remove `.then((res) => res)`

File: backend/src/main/resources/application.yml [hunk 2]
```diff
@@ -15,3 +15,6 @@ githubFeignClient:
 
 jira:
   url: https://demo.atlassian.net
+
+buildKite:
+  url: 54.223.108.141:4323
```
Comment (guzhongren) [inline backend/src/main/resources/application.yml:20]: Should not pass fixed IP into application.yaml
PLZ ref to https://stackoverflow.com/questions/23027315/does-application-yml-support-environment-variables to change

**Review by guzhongren — COMMENTED** [review]
Change with the comments

**Pull request merged**